python/core/auto_generated/qgsapplication.sip.in

```diff
@@ -381,6 +381,13 @@ Returns the path to user's style.
     static QRegExp shortNameRegExp();
 %Docstring
 Returns the short name regular expression for line edit validator
+%End
+
+    static QRegularExpression shortNameRegularExpression();
```

Comment:
Thinking about it -- do we even need to expose this? Seems like an internal detail to me

Reply:
I made an alt to keep the api stale. I highly doubt its used externally though..